modules/update-network-sysctl-allowlist.adoc

```diff
@@ -0,0 +1,167 @@
+// Module included in the following assemblies:
+//
+// * nodes/containers/nodes-containers-sysctls.adoc
+
+:_content-type: PROCEDURE
+[id="updating-interface-specific-safe-sysctls-list_{context}"]
+= Updating the interface-specific safe sysctls list
+
+{product-title} includes a predefined list of safe interface-specific `sysctls`. Modify this list by updating the `cni-sysctl-allowlist` in the `openshift-multus` namespace.
+
+[IMPORTANT]
+====
+The support for updating the interface-specific safe sysctls list is a Technology Preview feature only. Technology Preview features are not supported with Red Hat production service level agreements (SLAs) and might not be functionally complete. Red Hat does not recommend using them in production. These features provide early access to upcoming product features, enabling customers to test functionality and provide feedback during the development process.
+
+For more information about the support scope of Red Hat Technology Preview
+features, see https://access.redhat.com/support/offerings/techpreview/.
+====
+
+Follow this procedure to modify the predefined list of safe `sysctls`. This procedure describes extending the default allow list, the removal of `sysctls` is also supported.
```

Comment:
s/describes extending/describes how to extend ??

Reply:
> the removal of `sysctls` is also supported

Should you describe how to do this and/or show an example? Should the user remove a sysctls rather than comment it out? Does it make a difference?